### DISCUSSION TITLE
Does the install repo need updating to reflect docker changes?

### DISCUSSION THREAD

**Discussion by devchris2303** [postalserver/postal#2777] [Help with using Postal] [answered]
I'm trying to get a new instance up and running but either the documentation needs updating or the instal repo needs updating.

Attached is the full output from creating the blank VM to the error - nothing is ommitted.

I believe there is an issue with the install repo still using docker-compose rather than the new "docker compose"?

[postal install output.txt](https://github.com/postalserver/postal/files/14138799/postal.install.output.txt)

**Comment by willpower232** [comment, marked as answer]
![image](https://github.com/postalserver/postal/assets/1619102/d2cb0dff-f368-4a60-95de-65ecfdf155a4)

see #2774

**Reply by devchris2303** [reply]
Champion, thanks for this. 

I updated the docker-compose.yml file to pull the 2.1.6 image(s) and that solved it.
